frontend/src/components/SamplesList.tsx

```diff
@@ -27,30 +32,28 @@ interface ISampleListProps {
   columnDefs: ColDef[];
   defaultColDef: ColDef;
   getRowData: (samples: Sample[]) => any[];
-  height: number;
-  setUnsavedChanges?: (val: boolean) => void;
-  searchVariables?: SampleWhere;
-  filter: (searchVal: string) => SampleWhere;
+  setUnsavedChanges?: (unsavedChanges: boolean) => void;
+  parentWhereVariables?: SampleWhere;
+  refetchWhereVariables: (parsedSearchVals: string[]) => SampleWhere;
```

Comment:
This prop was previously named "filter".

We had many props that take in a GraphQL Where variable with different names. I'm renaming them all to include substring `WhereVariables` to standardize them.

Here, the `refetch` part refers to the fact that this prop gets used when we call the GraphQL `refetch()` function below.